frappe/geo/utils.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Frappe Technologies and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+
+import frappe
+
+from pymysql import InternalError
+
+
+
+
+@frappe.whitelist()
+def get_coords(doctype, filters, type):
+    '''Get list of coordinates in form
+    returns {names: ['latitude', 'longitude']} or location type'''
```

Comment:
As mentioned in https://github.com/frappe/frappe/pull/11202#issuecomment-670602373, `get_coords` should always return a geojson string. This will make display and maintenance much easier if we have one return type instead of 2.
```suggestion
    returns {name, location} with location being a geojson string'''
```